README.md

```diff
@@ -121,6 +121,14 @@ A plugin that renders the interface over the video container and add the possibi
 
 ### Click to pause
 Adds the possibility to toggle between the `play`/`pause` playback states by clicking on the container element.
+```javascript
+var player = new Clappr.Player({
+  source: "http://your.video/here.mp4",
+  // not mandatory, use only if you need it
+  clickToPauseConfig: { 
+    onClickPayload: { any: 'any' } // sends the payload to container when clicked
+});
+```
```

Comment:
I think it'd be cool to change the "not mandatory" comment to: 
```
Optionally, send a payload upon the container's pausing with the `onClickPayload` parameter.
```

Similar to the documentation on the Google Analytics plugin down below.

Reply:
Nice! Thanks @pedrochamberlain !